source/prerequisites/postgres.txt

```diff
@@ -1,13 +1,147 @@
 .. _rm-prereq-postgres:
 
-================================
-Postgres Migration Prerequisites
-================================
+==================================
+PostgreSQL Migration Prerequisites
+==================================
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Postgres.
\ No newline at end of file
+To run sync jobs from an PostGreSQL source database, the database may 
+require some configuration changes. If Relational Migrator determines the 
+database needs configuration changes, it automatically generates a 
+SQL script with the required changes. It is recommended to have a 
+Database Administrator (DBA) review the commands in this script and 
+perform their execution on the database server. These instructions 
+configuration PostGreSQL for both types of sync jobs:
+
+.. include:: /includes/fact-short-sync-job-desc.rst
+
+About this Task
+---------------
+
+Relational Migrator uses the PostgreSQL ``pgoutput`` plugin for logical 
+decoding. This plug-in is included in PostgreSQL versions 10+, no 
+additional installation or configuration is required.
+
+Before you Begin
+----------------
+
+If PostgreSQL is configured as a cluster, Relational Migrator must 
+connect to the master server.
+
+Steps
+-----
+
+.. tabs::
+
+   .. tab:: Snapshot Jobs
+      :tabid: enable-snapshot-jobs
+
+      For snapshot jobs against PostGreSQL, the service account requires
+      schema ``USAGE`` and table ``SELECT`` permissions. Replace 
+      ``POSTGRES`` with the name of your service account user:
+
+      .. code-block:: sql
+         :copyable: true
+
+         GRANT USAGE ON SCHEMA SCHEMA_NAME TO POSTGRES;
+         GRANT SELECT ON TABLE SCHEMA_NAME.TABLE_NAME TO POSTGRES;
```

Comment:
I confirmed with Poonam that schema usage and select is required even for snapshot: https://github.com/mongodb-ets/migrator/blob/main/application/src/main/resources/script/postgresql.mustache#L35-L42.

Reply:
OK. One thing to note is that Postgres is really funny about names that aren't all lowercase. While I know it's an example, can we change the sample code to:
```
GRANT USAGE ON SCHEMA <schema_name> TO <database_user_name>;
GRANT SELECT ON TABLE <schema_name>.<table_name> TO <database_user_name>;
```
This also makes it more obvious what needs to be changed.

Reply:
Updated.